README.md

```diff
@@ -176,9 +176,34 @@ To support [federated tracing](https://www.apollographql.com/docs/apollo-server/
     end
     ```
 
+## Exporting the Federated SDL
+
+When using tools like [rover](https://www.apollographql.com/docs/rover/) for schema validation, etc., add a Rake task that prints the Federated SDL to a file: 
+
+```rb
+namespace :graphql do
+  namespace :federation do
+    task :dump do
+      File.open "schema.graphql", "w+" do |f|
+        f << MySchema.federation_sdl
+      end
```

Comment:
```suggestion
      File.write("schema.graphql", MySchema.federation_sdl)
```

Reply:
Since this is an example rake task, suggesting to use `File.write` to keep it as simple as possible.